python/src/dataset.rs

```diff
@@ -2222,10 +2222,13 @@ impl PyFullTextQuery {
     }
 
     #[staticmethod]
-    #[pyo3(signature = (query, column))]
-    fn phrase_query(query: String, column: String) -> PyResult<Self> {
+    #[pyo3(signature = (query, column, slop=0))]
```

Comment:
Minor nit: the default `=0` isn't needed here since you always pass in a value (`self._inner = PyFullTextQuery.phrase_query(query, column, slop)`)